standalone.go

```diff
@@ -46,7 +46,7 @@ func doStandaloneInstall(device *deviceManager, args runOptionsType,
 	var upclient client.Updater
 
 	if args == (runOptionsType{}) {
-		return errors.New("rootfs called without needed parameters")
+		return errors.New("Image file called without needed parameters")
```

Comment:
The original intention here was to point to the name of the command line option, which was `rootfs`. I think we should change it to `install`, so that it still corresponds to the CLI.